### PR TITLE
use self.device

### PR DESCRIPTION
**Pull request by pravdomil**
# What does this PR do?

fixes rerender_a_video.py pipeline to use self.device instead of cuda

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/huggingface/diffusers/blob/main/CONTRIBUTING.md)?
- [ ] Did you read our [philosophy doc](https://github.com/huggingface/diffusers/blob/main/PHILOSOPHY.md) (important for complex PRs)?
- [ ] Was this discussed/approved via a GitHub issue or the [forum](https://discuss.huggingface.co/c/discussion-related-to-httpsgithubcomhuggingfacediffusers/63)? Please add a link to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/diffusers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/diffusers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

@SingleZombie

### CODE REVIEW

File: examples/community/rerender_a_video.py [hunk 1]
```diff
@@ -320,6 +321,7 @@ def __init__(
             image_encoder,
             requires_safety_checker,
         )
+        self.to(device)
```
Comment (DN6) [inline examples/community/rerender_a_video.py:324]: Don't this will work since the Pipeline isn't a `nn.Module`

**Review by DN6 — APPROVED** [review]
Nice 👍🏽

**Comment by pravdomil** [comment]
...

File: examples/community/rerender_a_video.py [hunk 2]
```diff
@@ -307,6 +307,7 @@ def __init__(
         feature_extractor: CLIPImageProcessor,
         image_encoder=None,
         requires_safety_checker: bool = True,
+        device=None,
```
Comment (yiyixuxu) [inline examples/community/rerender_a_video.py:310]: why do we need to this?
we can just do `pipe.to()` after we create the pipeline, no?

**Comment by pravdomil** [comment]
fixed

**Comment by pravdomil** [comment]
now should be ok

**Pull request merged**